File: Code/Lua/斐波那契数列.lua
function fac(n)
				if (n == 1) or (n == 2) then
								return 1
				else
								return fac(n - 1) + fac(n - 2)
				end
end

local number = io.input()

io.write(number, "is", fac(number))

File: Python/斐波那契数列.py
#!/usr/bin/python

def fac(n):
    if n == 1 or n == 2:
        return 1
    else:
        return fac(n - 1) + fac(n - 2)

if __name__ == "__main__":
    number = int(input())
    n = fac(number)
    print(n)
